tdrs-frontend/nginx/cloud.gov/buildpack.nginx.conf

```diff
@@ -0,0 +1,81 @@
+
+worker_processes 1;
+daemon off;
+
+error_log stderr;
+events { worker_connections 1024; }
+
+http {
+    charset utf-8;
+    log_format cloudfoundry 'NginxLog "$request" $status $body_bytes_sent';
+    access_log /dev/stdout cloudfoundry;
+
+    default_type application/octet-stream;
+    include mime.types;
+    sendfile on;
+
+    tcp_nopush on;
+    keepalive_timeout 30;
+    port_in_redirect off; # Ensure that redirects don't include the internal container PORT - 8080
+
+    log_format compression '$remote_addr - $remote_user [$time_local] '
+                            '"proxy_host and upstream_addr": $proxy_host $upstream_addr, '
+                           ' "request": $request, '
+                           '"body_bytes_sent" : $body_bytes_sent, ' 
+                           '"request_body": $request_body, '
+                           '"http_x_forwarded_for": $http_x_forwarded_for, ' 
+                            '"host": $host, '
+                            ' "status": $status, '
+                            '"proxy_add_x_forwarded_for": $proxy_add_x_forwarded_for, '
+                           '"http_referer": $http_referer, '
+                           '"http_user_agent": $http_user_agent, '
+                           '"cookies=$http_cookie;"  "server=$server_name" "http_host=$http_host"';
+
+    limit_req_zone $binary_remote_addr zone=limitreqsbyaddr:20m rate=10r/s;
```

Comment:
I think I commented on this elsewhere already but I think we might need to stress test or up the rate-limit here.

Reply:
We discussed this briefly and I think we should have to increase the limit. I have done some testing and got a rejection only once. But one caveat is: when frontend auth check function gets rejected, if user is already logged in, the backend will log them out.

Reply:
Increased the rate! Resolved